ddpui/management/commands/mark-connections-as-large-enough.py

```diff
@@ -0,0 +1,49 @@
+"""for each org fetch orchestration pipelines for airbyte sync"""
+
+from datetime import datetime
+import pytz
+from django.core.management.base import BaseCommand, CommandParser
+
+from ddpui.models.tasks import OrgTask
+from ddpui.utils.constants import TASK_AIRBYTESYNC, TASK_AIRBYTERESET, UPDATE_SCHEMA
+from ddpui.models.org import Org, ConnectionJob, ConnectionMeta
+from ddpui.ddpairbyte.airbyte_service import get_jobs_for_connection, abreq
+
+
+class Command(BaseCommand):
+    """Docstring"""
+
+    help = """This script is to figure out if the connection is large 
+    enough to schedule related schema change & reset jobs for later"""
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument("--org", type=str, help="org slug, optional", required=True)
+        parser.add_argument(
+            "--connection_id",
+            type=str,
+            help="connection_id, optional; if not passed all connections of the org will be considered",
+            required=False,
```

Comment:
let's make this a required parameter? the only time we would run the script without specifying a connection ID would be to create default `ConnectionMeta` objects, but the logic which accesses `schedule_large_jobs` works / should work without a `ConnectionMeta`